src/ert/_c_wrappers/enkf/runpaths.py

```diff
@@ -61,40 +60,28 @@ def write_runpath_list(
         The runpath list file is parsed by some workflows in order to find
         which path was used by each iteration and ensemble.
 
-        The following example code:
-
-            >>> runpath_file = "._ert_runpath_list"
-            >>> runpaths = Runpaths(
-            ...    "job%d",
-            ...    "realization-%d/iteration-%d",
-            ...    runpath_file,
-            ... )
-            >>> runpaths.write_runpath_list([0,1], [3,4])
-
-        Will result in runpath_file containing, when run with "/cwd/"
-        as current working directory:
-
-            003  /cwd/realization-3/iteration-0  job0  000
-            004  /cwd/realization-4/iteration-0  job0  000
-            003  /cwd/realization-3/iteration-1  job1  001
-            004  /cwd/realization-4/iteration-1  job1  001
+        Calling write_runpath_list([0,1], [3,4]) with "/cwd/" as the
+        current working directory will result in a runpath_file containing:
```

Comment:
The new text probably needs the context of what `job_fmt` and `runpath_fmt` is.

Reply:
Why? The context for `jobname_format` and `runpath_format` is present in the class documentation? Is there a reason to repeat what they are?

Reply:
The example assumes that `job_fmt` is `"JOB<IENS>"` and `runpath_format` is `"iteration_<ITER>/realization_<IENS>"`.

Reply:
added extra information